File: CMS/src/app/shared/DanhMucDuAn.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucDuAn } from 'src/app/shared/DanhMucDuAn.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucDuAnService extends BaseService{

    DisplayColumns001: string[] = ['STT', 'Name', 'Note', 'Save'];

    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucDuAn";
    }
}

File: Service/Implement/DanhMucDuAnService.cs
namespace Service.Implement
{
    public class DanhMucDuAnService : BaseService<DanhMucDuAn, IDanhMucDuAnRepository>
    , IDanhMucDuAnService
    {
        private readonly IDanhMucDuAnRepository _DanhMucDuAnRepository;
        public DanhMucDuAnService(IDanhMucDuAnRepository DanhMucDuAnRepository) : base(DanhMucDuAnRepository)
        {
            _DanhMucDuAnRepository = DanhMucDuAnRepository;
        }
    }
}

